testing/test_script.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+def test_script():
+    os.system("ivadomed_convert_to_onnx -m testing_data/model_unet_test.pt -d 2")
```

Comment:
I am only wondering if there is not a way to deal directly with pytest instead of doing a system call. And if so, is there a benefit? Just a comment, can be addressed in the future.

Reply:
I guess we could just call the function inside the script with the right parameters... Not sure if there is a benefit though.

Reply:
the point of this system call is also to make sure the argparse parameters are tested (e.g. scenario: a contributor decides to change the CLI argument, but not the API argument → test would pass)